src/components/BookDetail/BookDetail.tsx

```diff
@@ -198,7 +198,12 @@ export const BookDetailInternal: React.FunctionComponent<{
                             <DeleteButton book={props.book} />
                         </div>
                         {/* Enhance, maybe, add this and wire to some message <HowToPrintButton />*/}
-                        {bloomDesktopAvailable || (
+                        {/* This was previously and erroneously || instead of &&, which made it show up
+                        only when blook desktop is NOT available, that is, on mobile. Keeping it around
+                        because we might actually want it on desktop...BL-9201 is not clear. But, if so,
+                        we need a different, probably contentful, page and href, because the one below
+                        doens't work.
+                            {bloomDesktopAvailable && (

```

Comment:
I'm pretty sure the intent is for it to only show up on mobile. But we don't want it to show at all for the moment since it isn't implemented.